projects/kyc/Cargo.toml

```diff
@@ -18,7 +18,7 @@ codegen-units = 1 # Compile in one unit for better optimization and smaller bina
 panic = "abort"   # `no-std` can't unwind on panic, so choose abort (plus no panic handler means smaller binary)
 
 [profile.dev]
-panic = "unwind" # For debugging, support std-lib with unwinding in the `dev` profile
+panic = "abort"
```

Comment:
One reason to keep `unwind` for the dev profile is that if we ever execute code as Rust (e.g., a unit test, or a Rust simulation of the contract, not in WASM) then having the unwind is useful for development purposes. 

That said, if these Rust contracts only ever get compiled to WASM, then using `unwind` (even in the dev profile) doesn't really do much.

However, it doesn't hurt, and on the off chance that we do something in rust, it would be nice to have the unwind.

So, any objection to leaving the `dev` panic alone?